src/utils/keychain.js

```diff
@@ -38,10 +47,44 @@ export type KeyChainData = {
   pin?: ?string,
 };
 
+export const handleCatch = (accountAddress: ?string) => {
+  const colors = getThemeColors();
+  const buttons = [];
+  buttons.push({
+    text: t('error.failedKeychain.exitButtonText'),
+    onPress: () => RNExitApp.exitApp(),
+  });
+  accountAddress = accountAddress ?? etherspotService?.getAccountAddress(CHAIN.ETHEREUM);
+  if (accountAddress) {
+    buttons.push({
+      text: t('error.failedKeychain.supportButtonText'),
+      onPress: () => {
+        BugReporting.showWithOptions(BugReporting.reportType.feedback, [BugReporting.option.emailFieldOptional]);
+      },
```

Comment:
Could this be:

```suggestion
      onPress: BugReporting.showWithOptions(BugReporting.reportType.feedback, [BugReporting.option.emailFieldOptional]),
```

Reply:
yeah but since I wrote a test case, it will directly call this function and it will fail since the Instabug will not be initialised